### PR TITLE
docs: Missing backticks around identifiers.

### PR DESCRIPTION
**Pull request by waywardmonkeys**

### CODE REVIEW

File: wtransport/src/config.rs [hunk 1]
```diff
@@ -236,7 +236,7 @@ impl ServerConfigBuilder<WantsTransportConfigServer> {
     /// Keep-alive packets prevent an inactive but otherwise healthy connection from timing out.
     ///
     /// `None` to disable, which is the default. Only one side of any given connection needs keep-alive
-    /// enabled for the connection to be preserved. Must be set lower than the idle_timeout of both
+    /// enabled for the connection to be preserved. Must be set lower than the `idle_timeout` of both
```
Comment (waywardmonkeys) [inline wtransport/src/config.rs:239]: Should this actually be `idle_timeout` or should it be `max_idle_timeout`? (And same below...)

Comment (BiagioFesta) [reply wtransport/src/config.rs:239]: Yeah I guess `max_idle_timeout` makes more sense.

Probably ref to method to set, so it easier to navigate through the doc. That is, 
```
[`max_idle_timeout`](Self::max_idle_timeout)
```

what do you think?

Comment (waywardmonkeys) [reply wtransport/src/config.rs:239]: Agree, will do in a couple of hours when I'm back from badminton.

Comment (BiagioFesta) [reply wtransport/src/config.rs:239]: Thanks

Comment (waywardmonkeys) [reply wtransport/src/config.rs]: Updated. Also added a thing to enable these warnings in the regular clippy runs.

**Review by BiagioFesta — APPROVED** [review]

**Comment by BiagioFesta** [comment]
Thanks!

**Pull request merged**